### PR TITLE
WIP: docs updates for spring 2023 norns update

### PR DESCRIPTION
**Pull request by dndrks**
just a placeholder PR to collect feedback + ask q's :)

### CODE REVIEW

File: norns/reference/lib/timeline.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    0, {print, 'song starting'},
+    4, function() one:play() cutoffs:play() end,
+    9, function() one:stop() two:play() engine.release(2) end,
+    30, function() one:stop() two:stop() simple:stop() print('song ended') end
+  }
+
+end
+
+function send_hz(hz)
+  engine.hz(hz)
```
Comment (tehn) [inline norns/reference/lib/timeline.md]: highly suggest instead:

function send_hz(note)
  engine.hz(mu.note_num_to_freq(note))

that way you can remove a ton of typing upwards

Comment (dndrks) [reply norns/reference/lib/timeline.md]: oh duh, lol thank you!!

Comment (tehn) [reply norns/reference/lib/timeline.md]: dang that was fast

File: norns/play.md [hunk 2]
```diff
@@ -331,8 +331,26 @@ To clear the script running on norns:
...
+![](/docs/norns/image/play-images/system.png)  
+[*figure 9: the norns SYSTEM menu*](image/play-images/system.png)
+
+The SYSTEM menu contains a number of helpful entries:
+
+- `DEVICES` allows you to manage all your connected devices (MIDI, grid, arc, and HID). See [control + clock](/docs/norns/control-clock/) for more info.
+- `WIFI` manages network connectivity, including hotspot mode, wireless access, and ethernet. See [WiFi / update / files](/docs/norns/wifi-files/) for more info.
+- `MODS` are small chunks of code to create custom modifications to the core workings of the norns system software. See [mods](/docs/norns/community-scripts/#mods) for more info.
+- `SETTINGS` allows you to:
+  - `RESET` your norns back to its workshop settings, erasing your favorites, disabling all mods, and restoring global audio levels to their defaults.
```
Comment (pq) [inline norns/play.md]: do you mean _workshop_? (or maybe "default", "factory" or something?)

Comment (dndrks) [reply norns/play.md]: oh! lol i always opt for 'workshop' over 'factory', to try and set the right mental image of our operations, but will change to 'default', thank you!

**Comment by dndrks** [comment]
alright, i think that's it! everything from the [2.7.4 changelog](https://github.com/monome/norns/blob/main/update/changelog.txt#L3-L13) has been covered, except for the Lua NRT processing, which will require a bit more review before a doc update is possible

**Pull request merged**